.github/workflows/web_actions.yml

```diff
@@ -138,6 +142,25 @@ jobs:
         run: firebase deploy --only hosting:metrics-skia
         working-directory: metrics/web/
 
+      - name: Prepare metrics config for production environment
+        env:
+          SENTRY_RELEASE: ${{ env.SENTRY_RELEASE_AUTORENDERER }}
+          GOOGLE_SIGN_IN_CLIENT_ID: ${{ env.PRODUCTION_GOOGLE_SIGN_IN_CLIENT_ID }}
+        run: |
+          export SENTRY_ENVIRONMENT="production-skia"
```

Comment:
I wonder if we can define an environment variable instead of using the `export` command?